extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -36,8 +39,123 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
   transition: opacity 0.2s cubic-bezier(0, 0, 0.2, 1) !important;
 }
 
-/** Make the dialog wrapper from amp-subscriptions round corner. */
+/** Override the dialog styles from amp-subscriptions. */
 amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
-  border-radius: 16px 16px 0px 0px !important;
+  border-radius: 16px 16px 0 0 !important;
   font-family: Poppins, sans-serif !important;
+  font-weight: 400 !important;
+  background: white !important;
+  color: black !important;
+  overflow: hidden !important;
+}
```

Comment:
nit: Add a space between css rules. Please do this here and elsewhere in the document.